src/java.base/share/classes/java/nio/file/Path.java

```diff
@@ -516,6 +516,89 @@ default Path resolve(String other) {
         return resolve(getFileSystem().getPath(other));
     }
 
+    /**
+     * Resolves a path or a sequence of paths against this path.
+     *
+     * <p> The result of this method is the same as would be obtained
+     * by resolving the first parameter path against this path, then
+     * resovling the second parameter path against the derived path,
```

Comment:
Typo - should have been "resolving" instead of "resovling"

Reply:
Fixed in dd8c1413fc42b6b526d77c7131a10969353e0d08.